### PR TITLE
Liberating Structures Experinces Post

### PR DESCRIPTION
**Pull request by NiallBunting**
This aims to be a post that evolves over time as i add my thoughts and feelings about each structure.

### CODE REVIEW

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "Experiences of using liberating structures"
+date:   2020-06-10 13:20:00
+categories: liberating structures experiences
+---
+
+
+Liberating structures promise to unleash innovation, creativity and great ideas. I capture my thoughts and feelings, from my experiences with liberating  microstructures.
```
Comment (NiallBunting) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:9]: s/I capture/Here I capture/

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Liberating structures promise to unleash innovation, creativity and great ideas. I capture my thoughts and feelings, from my experiences with liberating  microstructures. 
+
+As part of this post I will include a brief overview of what these microstructures are, for more information about specific structures see the [liberating structure website][liberatingstructures].
+
+
+[liberatingstructures]: http://www.liberatingstructures.com/
+
+## What are Structures
+
+Structures are the various parts that come together to make a workplace/enterprise. These are split into both 'Macrostructures', 'Microstructures' and 'Structural elements'.
```
Comment (JPJDoran) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:18]: Probably remove 'both' here as it's a three part list.

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* Macrostructures are the larger and slower moving items such as the core operating model, buildings and strategy.
+* Microstructures are the smaller parts such as; meeting rooms, brainstorm and discussions. This is where and how discussions take place.
+* Structural elements facilitate the microstructures. For example, whiteboards, time allocations and agendas.
+
+
+All the structures play an important role, as each builds upon the smaller elements. Small changes on the lower levels over time can significantly effect the bigger structures. I have noticed for example, that virtual meetings seem to have a smaller pool of ideas and converge to a single idea much faster compared to in person meetings. Possibly, due to the fact virtual meetings only have one person speaking and it can be difficult to edge in. This reduces the pool of ideas and the best idea may not be chosen.
```
Comment (JPJDoran) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:24]: Perhaps change 'builds upon the smaller elements' to 'builds upon the other'

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+## Conventional Structures
+
+There are five main conventional structures that people use:
+* Presentations
+* Status Reports
+* Managed Discussions
+* Open Discussions
+* Brainstorms
+
+These are just the default options for information sharing, however they have a few problems as the first three are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. The final two (open discussions and brainstorms) are often too loose and disorganised to drive actual outcomes.
```
Comment (JPJDoran) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:37]: Suggested rewording:

"These are the most common options for information sharing, but they aren't without their problems. The first three structures are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. On the other hand, the final two are often too loose and disorganised to drive actual outcomes."

I'd suggest maybe including a linking sentence like "However, these problems can be solved through the use of liberating structures."

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+* Open Discussions
+* Brainstorms
+
+These are just the default options for information sharing, however they have a few problems as the first three are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. The final two (open discussions and brainstorms) are often too loose and disorganised to drive actual outcomes. 
+
+## Liberating Structures
+
+Liberating structures are 33 alternatives to the conventional structures noted above. They aim to structure your meeting to get everyone involved whilst not inhibiting anyone.
+
+The 33 have different use cases and often have different outcomes, so you picking the right tool for the job is required here. Many of them leverage the '1-2-4-All' structure, where you think alone, in pairs, in quads and then share back with the rest of the group.
```
Comment (JPJDoran) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:43]: Suggested rewording: 

"Each structure has its own different use cases and they often have different outcomes, so picking the right tool for the job is important. Many of them leverage the '1-2-4-All' structure, where you think alone, in pairs and in quads before then sharing with the rest of the group".

File: _posts/2020-06-10-experiences-using-liberating-structures.markdown [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+##### [Participant](#p)
+1. [Wicked Questions](#p-wicked-questions)
+-->
+
+
+#### <a name="f"/>Acted as Facilitator
+
+<ul>
+  <details>
+  <summary><a name="f-wicked-questions"/><li><strong>Wicked Questions</strong> - Used to find the opposing strategies that both need to be pursued to be successful.</li></summary>
```
Comment (JPJDoran) [inline _posts/2020-06-10-experiences-using-liberating-structures.markdown:69]: Perhaps add in here how to use the method too i.e. 

"Used to find the opposing strategies that both need to be pursued to be successful, by..."

**Pull request merged**